frontend/src/metabase/writeback/components/ActionViz/ActionViz.tsx

```diff
@@ -1,6 +1,10 @@
 import { t } from "ttag";
+import type { VisualizationSettings } from "metabase-types/api";
 import Action from "./Action";
 
+const isForm = (object: any, computedSettings: VisualizationSettings) =>
+  computedSettings.actionDisplayType === "form";
+
```

Comment:
Seems like we don't need the first argument here (`object`)?

Reply:
correct, but the settings framework gives it to us anyway